enterprise_gateway/mixins.py

```diff
@@ -431,6 +433,34 @@ def kernel_headers_default(self):
     def remote_hosts_default(self):
         return os.getenv(self.remote_hosts_env, self.remote_hosts_default_value).split(",")
 
+    # load_balancing_algorithm
+    load_balancing_algorithm_env = "EG_LOAD_BALANCE_ALGORITHM"
```

Comment:
Could we name this like the variable:
```suggestion
    load_balancing_algorithm_env = "EG_LOAD_BALANCING_ALGORITHM"
```